Sources/Testing/Test+Discovery.swift

```diff
@@ -103,11 +103,15 @@ extension Test {
       }
       let suiteID = ID(typeInfo: suiteTypeInfo)
       if tests[suiteID] == nil {
-        // If the real test is hidden, so shall the synthesized test be hidden.
-        // Copy the exact traits from the real test in case they someday carry
-        // any interesting metadata.
-        let traits = test.traits.compactMap { $0 as? HiddenTrait }
```

Comment:
Is this no longer needed?

Reply:
Yes, it's redundant as we already recursively apply traits like this one during planning.